pages/index.js

```diff
@@ -88,7 +88,7 @@ export default function Home() {
             <div className="md:flex md:justify-center">
               <MainDonationForm />
             </div>
-            <div className="md:grid md:grid-cols-3 md:gap-5 md:space-x-4 ">
+            <div className="md:grid md:grid-cols-3 md:gap-5 ">
```

Comment:
try

```javascript
<div className="md:grid md:grid-cols-3 md:gap-x-8 md:gap-y-5">
```

spacing should look like it was before

Reply:
Yes that does the trick, thanks!